MARKETING/PressReleaseDrafts/2023/draft.md

```diff
@@ -53,13 +53,12 @@ Many [software packages and open-source projects now support WoT](https://www.w3
 tools for Thing Description validation and construction as well as support for Thing runtimes and 
 Thing Description directories supporting discovery.
 
-Market adoption of the Web of Things is rapidly growing.
-The Web of Things has been adopted or is under consideration for adoption by
-several standards organizations, including 
+Market adoption of the Web of Things is gaining momentum, with notable products such as SayWoT! and Desigo CC from Siemens, the CGLL Platform from Takenaka, and interesting products from the startup world like MONAS and Agorà from VAIMEE systems for creating digital twins of cast-resign transformers and crops-terrains, and Krellian Cloud which provides real-time data analytics for smart buildings.
```

Comment:
```suggestion
Market adoption of the Web of Things is gaining momentum, with notable products such as SayWoT! and Desigo CC from Siemens and a building management platform from Takenaka.
Interesting products from startups are also appearing like MONAS and Agorà from VAIMEE systems for creating digital twins of cast-resin transformers, and in the agricultural domain for crops and terrains; and Krellian Cloud, which provides real-time data analytics for smart buildings.
```

Reply:
Some changes to avoid one big sentence.  We are confirming Takenaka's product name but in the meantime I have put in a generic "a building management platform".